src/node_api/mqtt/types.rs

```diff
@@ -171,9 +171,9 @@ impl Topic {
             r"^transactions/0x([a-f0-9]{64})/included-block$",
             // Output topics
             r"^outputs/([a-f0-9]{64})(\d{4})$",
```

Comment:
This one doesn't have a `basic` prefix like the core API?

Reply:
Nope there is no prefix here <https://studio.asyncapi.com/?url=https://raw.githubusercontent.com/iotaledger/tips/stardust-event-api/tips/TIP-0028/event-api.yml>

Reply:
But the Core API doesn't have a prefix too

Reply:
I meant the indexer sorry

Reply:
Hm not sure now if this endpoint returns all output ids, not only basic outputs?

Reply:
In https://github.com/iotaledger/inx-mqtt/commit/c06f2a2fffc0a825b06d8be38c1c874eb8893c7e it didn't get changed

Reply:
okay let's keep it then